### PR TITLE
Display paid and unpaid tickets in admin/tickets#show

### LINKED ISSUE #1190: admin/tickets#show
The paid column shows true even if not all of the tickets are paid (some paid, some not). That's misleading and a problem when it comes to the financial side of the tickets.

If a user purchased 2 tickets in 2 different occasions, and only 1 ticket is paid, an organizer should know how many tickets are paid, and how many are not.

### PR DESCRIPTION
**Pull request by shlok007**
Closes #1190 .

### CODE REVIEW

File: app/views/admin/tickets/show.html.haml [hunk 1]
```diff
@@ -19,6 +19,7 @@
         %th E-Mail
         %th Affiliation
         %th Paid
+        %th Unpaid
```
Comment (Ana06) [inline app/views/admin/tickets/show.html.haml]: I think that we don't need an unpaid column. I would render in the paid one how many tickets out of the total are paid.

I mean, if a user bought 3 tickets and paid 2 render: `2/3`

File: app/views/admin/tickets/show.html.haml [hunk 2]
```diff
@@ -33,5 +34,7 @@
             %td
               = buyer.affiliation
             %td
-              = @ticket.paid?(buyer)
```
Comment (Ana06) [inline app/views/admin/tickets/show.html.haml]: the `paid? `method is only used here, so you can remove it form the ticket model. You will also have to modify the tests.

File: app/models/ticket.rb [hunk 3]
```diff
@@ -23,6 +23,14 @@ def paid?(user)
     ticket_purchases.paid.by_user(user).present?
   end
 
+  def no_of_paid(user)
```
Comment (Ana06) [inline app/models/ticket.rb]: I would change this method to return an string with how many tickets out of the total are paid. I mean "2/3" if a user bought 3 tickets and paid 2.

And I am not sure if the name of the method is clear, what about `tickets_paid(user)`?

File: app/views/admin/tickets/show.html.haml (hunk 1 above)
Comment (shlok007) [inline app/views/admin/tickets/show.html.haml]: Thank you. I'll update it. 🙂

**Comment by Ana06** [comment]
LGTM :wink: 

Can you please squash the two commits into one: https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one ?

**Pull request merged**